File: EASY/ONBOARDING/python/onboarding.py
# game loop
while True:
    enemy_1 = raw_input()  # name of enemy 1
    dist_1 = int(raw_input())  # distance to enemy 1
    enemy_2 = raw_input()  # name of enemy 2
    dist_2 = int(raw_input())  # distance to enemy 2
    # Enter the code here
    if dist_1 < dist_2:
        print(enemy_1)
    else:
        print(enemy_2)

File: EASY/ONBOARDING/go/onboarding.go
package main

import "fmt"

func main() {
    for
    {
        // enemy1: name of enemy 1
        var enemy1 string
        fmt.Scan(&enemy1)

        // dist1: distance to enemy 1
        var dist1 int
        fmt.Scan(&dist1)

        // enemy2: name of enemy 2
        var enemy2 string
        fmt.Scan(&enemy2)

        // dist2: distance to enemy 2
        var dist2 int
        fmt.Scan(&dist2)

        if dist1 < dist2 {
            fmt.Println(enemy1)
        } else {
            fmt.Println(enemy2)
        }
    }
}